src/data/polynomial/ring_division.lean

```diff
@@ -160,6 +160,36 @@ begin
   rw [← nat_degree_mul hp₁ hq₂, ← nat_degree_mul hp₂ hq₁, h_eq]
 end
 
+variables [char_zero R]
+
+@[simp] lemma degree_bit0_eq (p : R[X]) : degree (bit0 p) = degree p :=
```

Comment:
I am perfectly happy with this lemma.

In informal maths, a great deal of results are stated with the assumption that the characteristic is not 2.  In this case, simply assuming `is_regular (2 : R)` is probably enough, but this is probably a discussion that is better left for a more global approach to "characteristic" assumptions in mathlib.